recipes/archicad-apidevkit/all/test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conan import ConanFile
+from conan.tools.build import cross_building
+from conans import CMake
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not cross_building(self):
+            bin_path = os.path.join("bin", "test_package")
```

Comment:
Is it possible to simplify it, like, running an executable from the package instead of building a project? As you can see, it requires a SDK which is not available.

Reply:
Unfortunately we do not have any runnable in the package. Should I comment out that part?